metadata.rb

```diff
@@ -5,7 +5,7 @@
 license 'Apache 2.0'
 description 'Installs/Configures rack_user'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '0.1.1'
+version '0.2.1'
```

Comment:
Should this be 0.2.0? We usually reset patch version when we bump minor or major.